### PR TITLE
Changed the event names from camelCase to kebab-case

### PR DESCRIPTION
**Pull request by royduin**
As of: https://vuejs.org/v2/guide/components-custom-events.html#Event-Names, event names should be kebab-case. Got issues with this with DOM templates.

### CODE REVIEW

**Review by bietkul — APPROVED** [review]
lgtm :+1:

**Review by bietkul — CHANGES_REQUESTED** [review]
@royduin This PR would break the events for existing users.
We should make that change with backward compatibility by emitting the events for both cases (kebab-case and camelCase). 
In v2.0 we'll remove the camelCase events.

**Comment by royduin** [comment]
I've re-added them 👍🏻

File: packages/vue/src/components/basic/DownShift.jsx [hunk 1]
```diff
@@ -358,11 +358,12 @@ export default {
 			Object.keys(stateToSet).map(key => {
 				// eslint-disable-next-line
 				this.isControlledProp(key)
-					? this.$emit(`${key}Change`, stateToSet[key])
+					? this.$emit(`${key}Change`, stateToSet[key]) && this.$emit(`${key}-change`, stateToSet[key])
```
Comment (bietkul) [inline packages/vue/src/components/basic/DownShift.jsx:361]: @royduin Can we remove `&&` operator and use `if else` condition to emit events?

**Review by bietkul — CHANGES_REQUESTED** [review]
@royduin A minor change is needed. The rest of the changes look good to me.

**Comment by bietkul** [comment]
I'm merging it so it can be out in today's version.

**Comment by royduin** [comment]
Thanks you! 💯

**Pull request merged**